code/modules/ruins/lavalandruin_code/elephantgraveyard.dm

```diff
@@ -0,0 +1,227 @@
+	if(opened == FALSE)
+		to_chat(user, "<span class='notice'>The ground here is too hard to dig up with your bare hands. You'll need a shovel.")
+	else
+		to_chat(user, "<span class='notice'>The grave has already been dug up.</span>")
+
+/obj/structure/closet/crate/grave/tool_interact(obj/item/S, mob/living/carbon/user)
+	if(user.a_intent == INTENT_HELP) //checks to attempt to dig the grave, must be done on help intent only.
+		if(opened == FALSE)
+			if(istype(S,cutting_tool) && S.tool_behaviour == TOOL_SHOVEL)
+				to_chat(user, "<span class='notice'>You start start to dig open \the [src]  with \the [S]...</span>")
+				if (do_after(user,20, target = src))
+					opened = TRUE
+					locked = TRUE
+					dump_contents()
+					update_icon()
+					SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "graverobbing", /datum/mood_event/graverobbing)
+					if(lead_tomb == TRUE && first_open == TRUE)
+						user.gain_trauma(/datum/brain_trauma/magic/stalker)
+						to_chat(user, "<span class='boldwarning'>Oh no, no no no, THEY'RE EVERYWHERE! EVERY ONE OF THEM IS EVERYWHERE!</span>")
+						first_open = FALSE
+					if(untouched == TRUE)
+						new /obj/effect/decal/remains/human(loc)
+						untouched = FALSE
+					return 1
+				return 1
+			else
+				to_chat(user, "<span class='notice'>You can't dig up a grave with \the [S.name].</span>")
+				return 1
+		else
+			to_chat(user, "<span class='notice'>The grave has already been dug up.</span>")
+			return 1
+
+	else if(user.a_intent != INTENT_HELP) //checks to attempt to remove the grave entirely.
+		if(opened == TRUE)
+			if(istype(S,cutting_tool) && S.tool_behaviour == TOOL_SHOVEL)
+				to_chat(user, "<span class='notice'>You start to remove \the [src]  with \the [S].</span>")
+				if (do_after(user,15, target = src))
+					to_chat(user, "<span class='notice'>You remove \the [src]  completely.</span>")
+					SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "graverobbing", /datum/mood_event/graverobbing)
+					deconstruct(TRUE)
+					return 1
+	return
+
+/obj/structure/closet/crate/grave/bust_open()
+	..()
+	opened = TRUE
+	update_icon()
+	dump_contents()
+	return
+
+/obj/structure/closet/crate/grave/lead_researcher
+	name = "ominous burial mound"
+	desc = "Even in a place filled to the brim with graves, this one shows a level of preperation and planning that fills you with dread."
+	icon = 'icons/obj/crates.dmi'
+	icon_state = "grave_lead"
+	lead_tomb = TRUE
+	first_open = TRUE
+
+/obj/structure/closet/crate/grave/lead_researcher/PopulateContents()  //ADVANCED GRAVEROBBING
+	..()
+	new /obj/effect/decal/remains/human(src)
+	new /obj/effect/decal/cleanable/blood/gibs/old(src)
+	new /obj/item/book/granter/crafting_recipe/boneyard_notes(src)
+
+/obj/item/book/granter/crafting_recipe/boneyard_notes
+	name = "The Complete Works of Lavaland Bone Architecture"
+	desc = "Pried from the lead archeologist's cold, dead hands, this seems to explain how ancient bone architecture was erected, long ago."
```

Comment:
Archeologist -> Archaeologist
Technically both are correct, but the latter would be preferable for this context
https://grammarist.com/spelling/archaeology-archeology/

Also remove that last comma